spec/lib/shanty/artifact_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'shanty/artifact'
+
+# All classes referenced belong to the shanty project
+module Shanty
+  RSpec.describe(Artifact) do
+    subject { Artifact.new('html', 'test', uri) }
+
+    describe('#localfile') do
```

Comment:
I'm going to slightly reorder and rename these describe blocks as these should actually be real method names, eg. `describe('#local?')` or `describe('.some_class_method')`.